sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryClientBuilder.java

```diff
@@ -219,6 +213,17 @@ public SchemaRegistryClientBuilder retryPolicy(RetryPolicy retryPolicy) {
         return this;
     }
 
+    /**
+     * Sets the service version to use.
+     *
+     * @param serviceVersion Service version.
+     * @return The updated instance.
+     */
+    public SchemaRegistryClientBuilder serviceVersion(SchemaRegistryVersion serviceVersion) {
```

Comment:
[APIview suggestion](https://apiview.dev/Assemblies/Review/be51f943baa1490597548e88861bdfd5?diffOnly=True&diffRevisionId=030d2cda43be4e6daefdfee2f97f563d#com.azure.data.schemaregistry.SchemaRegistryVersion): make is `serviceVersion(ServiceVersion)`

Reply:
Good catch!